src/types/timestamp.rs

```diff
@@ -1007,230 +973,212 @@ impl TimestampBuilder {
         }
         Ok(timestamp)
     }
+
+    /// Like [Self::build], but the fields provided for each time unit are understood
+    /// to be in UTC rather than in the local time of the specified offset (if there is one).
+    pub(crate) fn build_utc_fields_at_offset(
+        mut self,
+        offset_minutes: i32,
+    ) -> IonResult<Timestamp> {
+        self.fields_are_utc = true;
+        self.offset = Some(offset_minutes);
+        self.build()
+    }
 }
 
-/// Allows the user to set the `month` field on a builder that has already had its `year`
-/// field set. Or, if `Year` is the desired precision, they may build the [Timestamp] with an
-/// unknown offset instead.
-#[derive(Debug, Clone)]
-pub struct MonthSetter {
-    builder: TimestampBuilder,
+#[derive(Debug, Clone, Default)]
+pub struct Init;
+
+impl Default for TimestampBuilder<Init> {
```

Comment:
🗺️ Clippy was unhappy about having `new()` without implementing `Default`, so we have an implementation of `Default` _only_ for `TimestampBuilder<Init>`.